cvmfs/download.cc

```diff
@@ -1646,6 +1641,15 @@ void DownloadManager::Init(const unsigned max_pool_handles,
       SetProxyChain(string(getenv("http_proxy")), "", kSetProxyRegular);
     }
   }
+
+  const char *use_cert_var = getenv("CVMFS_USE_SSL_SYSTEM_CA");
```

Comment:
Let's move this to the `MountPoint` class where the download managers for the client is initialized.  This way, we'd have the option handling for the download manager more centralized.  As a follow-up to this PR, it might anyway be a good idea to check why `X509_CERT_DIR` and `IPV4_ONLY` parameters are checked directly in the download manager and not in the code that initialized it.

Reply:
No, this is wrong, we are not able to create the repository if we move this into the `MountPoint` as done with: https://github.com/cvmfs/cvmfs/pull/2588/commits/01799e9a69363f710439edc308d7680efad5f7f2

Reply:
Fixed! Was a problem with the Clone method that was not taking into consideration the new use of system CA